app/src/sharedTest/java/org/oppia/android/app/help/HelpFragmentTest.kt

```diff
@@ -470,6 +470,513 @@ class HelpFragmentTest {
+          recyclerViewId = R.id.license_list_fragment_recycler_view,
+          position = 0
+        )
+      ).perform(click())
+      onView(withId(R.id.help_multipane_options_title_textview)).check(
+        matches(
+          withText(R.string.license_name_0)
+        )
+      )
+      onView(withId(R.id.copyright_license_text_view)).check(
+        matches(
+          isDisplayed()
+        )
+      )
+    }
+  }
+
+  @Test
+  @Config(qualifiers = "sw600dp")
+  fun testHelpFragment_openLicenseText_tabletConfigChanged_licenseListIsDisplayed() {
+    launch<HelpActivity>(
+      createHelpActivityIntent(
+        internalProfileId = 0,
+        isFromNavigationDrawer = true
+      )
+    ).use {
+      onView(withId(R.id.help_fragment_recycler_view)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(1)
+      )
+      onView(
+        atPosition(
+          recyclerViewId = R.id.help_fragment_recycler_view,
+          position = 1
+        )
+      ).perform(click())
+      onView(
+        atPosition(
+          recyclerViewId = R.id.third_party_dependency_list_fragment_recycler_view,
+          position = 0
+        )
+      ).perform(click())
+      onView(
+        atPosition(
+          recyclerViewId = R.id.license_list_fragment_recycler_view,
+          position = 0
+        )
+      ).perform(click())
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.help_multipane_options_title_textview)).check(
+        matches(
+          withText(R.string.license_name_0)
+        )
+      )
+      onView(withId(R.id.copyright_license_text_view)).check(
+        matches(
+          isDisplayed()
+        )
+      )
+    }
+  }
+
+  @Test
+  @Config(qualifiers = "sw600dp")
+  fun testHelpFragment_openLicenseText_multipaneOptionsAreDisplayed() {
+    launch<HelpActivity>(
+      createHelpActivityIntent(
+        internalProfileId = 0,
+        isFromNavigationDrawer = true
+      )
+    ).use {
+      onView(withId(R.id.help_fragment_recycler_view)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(1)
+      )
+      onView(
+        atPosition(
+          recyclerViewId = R.id.help_fragment_recycler_view,
+          position = 1
+        )
+      ).perform(click())
+      onView(
+        atPosition(
+          recyclerViewId = R.id.third_party_dependency_list_fragment_recycler_view,
+          position = 0
+        )
+      ).perform(click())
+      onView(
+        atPosition(
+          recyclerViewId = R.id.license_list_fragment_recycler_view,
+          position = 0
+        )
+      ).perform(click())
+      onView(withId(R.id.help_multipane_options_back_button)).check(
+        matches(
+          withContentDescription(
+            retrieveHelpOptionTextViewContentDescription("copyright licenses list")
+          )
+        )
+      )
+      onView(withId(R.id.help_multipane_options_back_button)).check(
+        matches(withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE))
```

Comment:
Update here and at other places
` matches(withEffectiveVisibility(Visibility.VISIBLE))`

Reply:
Done.